ibm/resource_ibm_database.go

```diff
@@ -1135,7 +1135,7 @@ func resourceIBMDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
 		if err != nil {
 			return fmt.Errorf("Error updating database scaling group: %s", err)
```

Comment:
`"Error updating database scaling group: %s"` ==> `""Error updating database cpu auto scaling group: %s"`

Reply:
Can you please make similar changes to 
`"auto_scaling.0.disk"`, `"auto_scaling.0.memory"`